### PR TITLE
Updated participants.json

### PR DESCRIPTION
**Pull request by barney-stinson**

### CODE REVIEW

**Comment by aswinzz** [comment]
@barney-stinson Resolve the merge conflicts

**Comment by barney-stinson** [comment]
Whats this merge conflict?

File: data/participants.json [hunk 1]
```diff
@@ -110,7 +110,17 @@
             "github"   : "http://github.com/coderjolly",
             "twitter"  : "http://twitter.com/coderjolly",
             "referral" : ""
-        },
+        }
```
Comment (shreyanshdwivedi) [inline data/participants.json]: You are missing a comma(,) here. This makes this json file invalid. You can test if your json code is valid or not, using https://jsonlint.com/

Comment (barney-stinson) [reply data/participants.json]: thank U :)

**Pull request merged**